@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -945,8 +1040,8 @@ export class TileGeometryCreator {
 
                     FadingFeature.addRenderHelper(
                         outlineObj,
-                        fadingParams.lineFadeNear,
-                        fadingParams.lineFadeFar,
+                        fadingParams.fadeNear,
+                        fadingParams.fadeFar,
```

Comment:
In this block we are creating lines mesh for polygon's outline, so all parameters are dedicated to lines mesh. Moreover, ```fadingParams.lineFadeNear``` is equal to ```fadingParams.fadeNear``` if no special property on technique is set.

Reply:
Your are right. Reverting this change.